tfe/resource_tfe_organization_token_test.go

```diff
@@ -99,6 +99,52 @@ func TestAccTFEOrganizationToken_existsWithForce(t *testing.T) {
 	})
 }
 
+func TestAccTFEOrganizationToken_existsWithoutExpiry(t *testing.T) {
+	token := &tfe.OrganizationToken{}
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+	orgName := fmt.Sprintf("tst-terraform-%d", rInt)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckTFEOrganizationTokenDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTFEOrganizationToken_basic(rInt),
```

Comment:
❓ was this meant to sue the new `testAccTFEOrganizationToken_existsWithoutExpiry`?

Reply:
Yup! 🤦‍♀️